core/templates/pages/topic-editor-page/topic-editor-page.component.html

```diff
@@ -17,7 +19,7 @@
           <span class="oppia-editor-warnings-count">
             <[$ctrl.getTotalWarningsCount()]>
           </span>
-          <ul uib-dropdown-menu ng-if="$ctrl.warningsAreShown" class="uib-dropdown-menu oppia-editor-warnings-box oppia-editor-warnings-box-block">
+          <ul uib-dropdown-menu ng-if="$ctrl.warningsAreShown" class="uib-dropdown-menu topic-warning-container oppia-editor-warnings-box oppia-editor-warnings-box-block">
```

Comment:
Why do we need two separate classes "topic-warning-container" and "oppia-editor-warnings-box-block"?

Reply:
Removed the oppia-editor-warnining-box-block.